packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -6,7 +6,8 @@ import { KeyboardSpinDirection } from './SpinButton';
 import { IButtonStyles } from '../../Button';
 import { IKeytipProps } from '../../Keytip';
 import { IRefObject } from '../../Utilities';
-import { IButtonProps } from '../Button/Button.types';
+//import { IButtonProps } from '../Button/Button.types';
```

Comment:
This is a breaking change. I am ok coordinating this change in react-next, but not in v7.